app/views/admin/registrations/index.html.haml

```diff
@@ -44,24 +44,15 @@
               %td
                 = link_to 'Questions','#', class: 'btn btn-success question-btn', 'data-id' => index, 'data-name' => registration.name
             %td
+              = check_box_tag "#{@conference.short_title}_#{registration.id}", registration.id, registration.attended,
+                class: 'switch-checkbox', method: :patch,
+                url: toggle_attendance_admin_conference_registration_path(@conference.short_title, id: registration.id)+"?attended=",
+                data: { size: "large",
```

Comment:
You should use size: 'small' like in #693 discussed ... Large still looks a little bit ugly :(